README.md

```diff
@@ -132,6 +170,26 @@ cratedb-about list-questions
 To configure a different context file, use the `CRATEDB_CONTEXT_URL` environment
 variable. The default value is https://cdn.crate.io/about/v1/llms-full.txt.
 
+## FAQ
+
+- Q: Seriously, how do I use this?
+
+  A: As mentioned above, this repository includes content and a few utilities
+  to manage corresponding information. Users will directly use the produced
+  [llms.txt] and [llms-full.txt] files. Developers will install the [cratedb-about]
+  package to access fundamental outline information in their own programs
+  programmatically, or to invoke fragments of the production machinery
+  on their premises, either ad hoc, or by including it in automated pipelines.
+
+- Q: It looks like the knowledge base machinery is missing important information
+  about CrateDB. I've asked it about matters of polymer sharding, and the answer
+  wasn't very insightful.
+
+  A: Well, we can understand your disappointment. To improve the situation,
+  we are constantly curating content, and you can support the process by giving
+  us hints about which fragments of information to include in the set of
+  curated information. To learn about what this means, see also [ABOUT-24].
```

Comment:
That paragraph has been added to **evangelize the need for support** about further content curation.

- GH-24

/cc @kneth, @michaelkremmel